### PR TITLE
elevation service one point per user test case

### PR DESCRIPTION
**Pull request by shyeavr**
Add graph and create one point per user logic

### CODE REVIEW

File: playgorund/playground.py [hunk 1]
```diff
@@ -1,27 +1,91 @@
...
 
-class MyUser(HttpUser):
-    wait_time = between(1, 3)
+# if type(ElevationConfig.percent_ranges_counters) == str:
+#     percent_ranges = json.loads(ElevationConfig.percent_ranges_counters)
+# else:
+# percent_ranges = ElevationConfig.ranges["ranges"]
+ranges = {
+    "ranges": [
+        {"lower": 0, "upper": 100},
```
Comment (dannysigalovich) [inline playgorund/playground.py:19]: @shyeavr  should be constants ?

File: playgorund/playground.py [hunk 2]
```diff
@@ -1,27 +1,91 @@
...
+    total_requests = 0
 
-# Start the Prometheus metrics server
-start_http_server(8080)
 
-# # Run Locust
-# MyUser().run()
+# Run the Locust test
+class MyUser(CustomUser):
+    min_wait = 100
```
Comment (dannysigalovich) [inline playgorund/playground.py:84]: @shyeavr Constants

File: tests/elevation_service/graph_function.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        plt.plot(users, avg_response_times, marker='o')
+        plt.ylabel('Average Response Time')
+        plt.xlabel('Number of Users')
+        plt.title("User amount vs Average Response Time")
+        plt.grid(True)
+        # plt.plot(avg_response_times, users, marker='o')
+        # plt.xlabel('Average Response Time (ms)')
+        # plt.ylabel('User Count')
+        # plt.title('User Count vs. Average Response Time')
+        plt.savefig('graph.png')
```
Comment (dannysigalovich) [inline tests/elevation_service/graph_function.py:72]: @shyeavr  check if exists or its override?

File: tests/elevation_service/elevation_test.py [hunk 4]
```diff
@@ -1,104 +1,82 @@
...
-    counter_range_2 = 0
-    counter_range_3 = 0
+    global counters, total_requests
+    counters = counters
     total_requests = 0
 
 
 # Run the Locust test
 class MyUser(CustomUser):
     min_wait = 100
```
Comment (dannysigalovich) [inline tests/elevation_service/elevation_test.py:80]: @shyeavr  Constants?

File: tests/elevation_service/elevation-integ.py [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+@events.test_start.add_listener
+def reset_counters(**kwargs):
+    global counters, total_requests
+    counters = counters
+    total_requests = 0
+
+
+# Run the Locust test
+class MyUser(CustomUser):
+    min_wait = 100
```
Comment (dannysigalovich) [inline tests/elevation_service/elevation-integ.py:78]: @shyeavr  constants?

**Review by dannysigalovich — CHANGES_REQUESTED** [review]
Look at the comments

File: tests/elevation_service/graph_function.py (hunk 3 above)
Comment (shyeavr) [reply tests/elevation_service/graph_function.py]: should be overridden for each test run to enable scenario runs analyzing

**Pull request merged**